doc/howto/instances_manage.md

```diff
@@ -76,3 +76,28 @@ There are two ways to prevent accidental deletion of instances:
 
 - To protect a specific instance from being deleted, set [`security.protection.delete`](instance-options-security) to `true` for the instance.
   See {ref}`instances-configure` for instructions.
+
+## Rebuild an instance
+
+If you want to wipe the file system of your instance but keep the instance configuration, you can rebuild the instance.
+
+Rebuilding is only possible for instances that do not have any snapshots.
+The reason for this limitation is that rebuilding an instance would also require rebuilding all snapshots, which would need advanced logic for each storage driver (because the storage drivers use very different methods for implementing snapshots).
+Therefore, if your instance has snapshots, you must delete the snapshots before you can rebuild the instance.
+
+Stop your instance before rebuilding it.
+Then enter one of the following commands:
+
+- Rebuild the instance with its original image (the one that was used to create the instance):
```

Comment:
Should we add something like "Or if the instance was created as empty, it will be rebuilt as empty"?

Reply:
sure ;)